src/testFixtures/groovy/grails/plugin/geb/ContainerGebSpec.groovy

```diff
@@ -72,6 +72,10 @@ abstract class ContainerGebSpec extends Specification implements ManagedGebTest,
 
     @PackageScope
     void initialize() {
+        if(webDriverContainer) {
```

Comment:
Space after if? I like the `notInitialized` condition better. It extracts the responsibility of that logic and makes the code more readable.

Reply:
What are your thoughts about going ahead and moving all of this to the spock extension?  The reason I in-lined it was to reduce the amount of logic in the `ContainerGebSpec`